docs/lesson-plans/week-1.md

```diff
@@ -262,9 +270,14 @@ Inside of IntelliJ, go to File > New... > Project From Version Control..., make
 `Version Control` box is set to `Git`, paste in the following links, and choose where you 
 want them to be downloaded to. 
 
-The links you'll want to use are 
-- Backend: https://github.com/Code-4-Community/jumpstart-backend/tree/starter-code
-- Frontend: __*GET STARTER CODE TAG DONE*__
+The links you'll want to use are in the __Important Links__ section above.
+
+After the projects are open, you'll want to view the starter code by entering the following command into the terminal
+at the bottom of your IntelliJ window for both the frontend and backend projects.
+
+```shell script
+git checkout starter-code
```

Comment:
We may want to give some brief context as to what "checkout" means, in general I think we should explain as much as is practical when it comes to any commands or code that we tell readers to enter.

Reply:
I think that we can hold off on explaining. Just make sure it works now. Git will be next week

Reply:
I just put a quick description in (definitely not enough to explain it though